### PR TITLE
Css ecommerce page

### PR DESCRIPTION
**Pull request by alexdenbaugh**
css-eccomerce-page

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! A couple visual fixes are needed, though:
1. The item's size should not be struck through, just the old price for the item.
2. The background color for the current price doesn't match the color of the example. Try something like "peach" for the color.

Once these are addressed, I will take a deeper look at this. Make sure you attach an updated screenshot when you push up your changes!

**Comment by alexdenbaugh** [comment]
css-eccomerce-page-revised

File: css-ecommerce-page/styles.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  margin: auto;
+  width: 1000px;
+  justify-content: center;
+}
+
+.row {
+  display: flex;
+  flex-basis: 100%;
+
+  /* width: 1000px; */
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css:18]: Please remove commented code before submitting

File: css-ecommerce-page/styles.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  flex-direction: row;
+  justify-content: space-evenly;
+  margin: 0;
+
+  /* border: black thin solid; */
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css:23]: Please remove commented code before submitting

File: css-ecommerce-page/styles.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.column {
+  display: flex;
+  flex-wrap: wrap;
+  width: 200px;
+  margin: 0 5px;
+  align-content: flex-end;
+
+  /* border: red thin solid; */
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css:33]: Please remove commented code before submitting

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice revisions!! A couple more minor things to adjust:

**Review by codingcodymiller — APPROVED** [review]
Nice work!!! 🎉

**Pull request merged**